apps/explorer/lib/explorer/chain.ex

```diff
@@ -6953,24 +6953,8 @@ defmodule Explorer.Chain do
 
   def get_implementation_abi_from_proxy(proxy_address_hash, abi)
       when not is_nil(proxy_address_hash) and not is_nil(abi) do
-    implementation_method_abi =
-      abi
-      |> Enum.find(fn method ->
-        Map.get(method, "name") == "implementation" ||
-          master_copy_pattern?(method)
-      end)
-
-    if implementation_method_abi do
-      implementation_address_hash_string = get_implementation_address_hash(proxy_address_hash, abi)
-
-      if implementation_address_hash_string do
-        get_implementation_abi(implementation_address_hash_string)
-      else
-        []
-      end
-    else
-      []
-    end
+    implementation_address_hash_string = Chain.get_implementation_address_hash(proxy_address_hash, abi)
```

Comment:
Why did you add `Chain` alias? `get_implementation_address_hash/2` method is available without it
```suggestion
    implementation_address_hash_string = get_implementation_address_hash(proxy_address_hash, abi)
```